jackal_base/CMakeLists.txt

```diff
@@ -8,6 +8,8 @@ find_package(catkin REQUIRED COMPONENTS
 find_package(Boost REQUIRED COMPONENTS thread)
 
 catkin_package()
+catkin_add_env_hooks(50.jackal_find_mag_config
```

Comment:
It's a feature which is extremely ripe for abuse, hence it not being overly publicized. But it's used in a number of packages to provide things like shell aliases, completions, etc.